lib/__tests__/build-files.test.js

```diff
@@ -56,9 +56,9 @@ beforeAll(() => {
   });
 });
 
-function afterAll() {
+afterAll(() => {
```

Comment:
This changes the meaning - It was declaring a function before, now it's executing it. The after version looks more correct though.

Reply:
yup. it's actually caught by the linter. Otherwise the function is never used